src/libs/config.js

```diff
@@ -9,7 +9,8 @@ export const getConfig = () => {
 }
 
 export const isAnvil = () => (window.location.hostname === 'anvil.terra.bio') || getConfig().isAnvil
-export const isAnalysisTabVisible = () => false || getConfig().isAnalysisTabVisible
+export const isAnalysisTabVisible = () => getConfig().isAnalysisTabVisible
```

Comment:
This doesn't seem to be used. Is this change necessary?

Reply:
I see it used in a few places, like workspacecontainer.js